### DISCUSSION TITLE
Where is the configure app?

### DISCUSSION THREAD

**Discussion by strjbyu** [ImageMagick/ImageMagick#7531] [Help]
### ImageMagick version

7.1.1-Q16

### Operating system, version and so on

Windows Server 2019

### Description

https://imagemagick.org/script/security-policy.php mentions:

o help you get started, as of version 7.1.1-16, ImageMagick provides security polices **that you can select when installing** [ ImageMagick](https://imagemagick.org/script/security-policy.php#). Choose from:

Then later it mentions: "Under Windows, the choice is presented when you run the **configure app**."

Where is the configure app located (and/or how it launched)? I'm running 7.1.1-Q16

**Comment by dlemstra** [comment]
Are you planning to compile your own version of ImageMagick or do you just want to use a different configuration?

**Reply by strjbyu** [reply]
I just want to use one of those standard security configurations (not compile my own). I didn't see any options for security configuration when I ran the installer.

**Comment by dlemstra** [comment]
You can then use one of the `policy-*.xml` files from this folder: https://github.com/ImageMagick/ImageMagick/tree/main/config and rename it to `policy.xml` and copy that to your installation directory.
